cmd/ocsp-updater/main.go

```diff
@@ -624,14 +624,9 @@ func (updater *OCSPUpdater) missingReceiptsTick(ctx context.Context, batchSize i
 		// If the feature flag is enabled, only send the certificate to the missing
 		// logs using the `SubmitToSingleCT` endpoint that was added for this
 		// purpose
-		if features.Enabled(features.ResubmitMissingSCTsOnly) {
-			for _, log := range missingLogs {
-				_ = updater.pubc.SubmitToSingleCT(ctx, log.uri, log.key, cert.DER)
-			}
-		} else {
-			// Otherwise, use the classic behaviour and submit the certificate to
-			// every log to get SCTS using the pre-existing `SubmitToCT` endpoint
-			_ = updater.pubc.SubmitToCT(ctx, cert.DER)
+
```

Comment:
Can you remove the whitespace between the comment and the start of your `if`? Thanks!